pyproject.toml

```diff
@@ -1,10 +1,10 @@
 [project]
 name = "cowdao-cowpy"
-version = "1.0.0pre1"
+version = "0.0.2"
 
 [tool.poetry]
 name = "cowdao-cowpy"
-version = "1.0.0pre1"
+version = "0.0.2"
 description = ""
 authors = ["José Ribeiro <jose@bleu.studio>"]
```

Comment:
feel free to add yourself to the authors!

Reply:
Oh man, that is super lovely thanks, ill do so now.